examples/big-data-section/tests/test_big_data_section.rs

```diff
@@ -0,0 +1,56 @@
+#[cfg(test)]
+mod tests {
+    use demo_big_data_section::data_access::DataAccess;
+    use gtest::{Log, Program, System};
+
+    const USER_ID: u64 = gtest::constants::DEFAULT_USER_ALICE;
+
+    // Testing random access to data section
+    #[test]
+    fn test_big_data_section() -> Result<(), &'static str> {
+        let sys = System::new();
+        sys.init_logger();
+
+        let prog = Program::from_file(
+            &sys,
+            "../../target/wasm32-unknown-unknown/debug/demo_big_data_section.opt.wasm",
+        );
+        sys.mint_to(gtest::constants::DEFAULT_USER_ALICE, 100000000000);
+
+        // Skipping program initialization
+        let _ = prog.send_bytes(USER_ID, b"");
+        let _ = sys.run_next_block();
+
+        let random_payload: Vec<Vec<u8>> = vec![
```

Comment:
could you pls use here proptests to make sure its not coincidence? also run a few times locally with huge amount of repetitions